src/controller/proxy/remote.go

```diff
@@ -73,6 +74,16 @@ func (r *remoteHelper) init() error {
 	}
 	adp, err := factory.Create(reg)
 	if err != nil {
+		if reg.Status == model.Healthy {
```

Comment:
The factory has different implementations so it returns error for various reasons, marking the registry unhealthy for whatever error doesn't seem correct to me.
How it will impact if we just leave it as is?
@ywk253100  thoughts?

Reply:
remove it